enterprise/backend/src/metabase_enterprise/representations/v0/core.clj

```diff
@@ -0,0 +1,77 @@
+(ns metabase-enterprise.representations.v0.core
+  (:require
+   [metabase-enterprise.representations.toucan.core :as rep-t2]
+   [metabase-enterprise.representations.v0.collection :as v0-coll]
+   [metabase-enterprise.representations.v0.common :as v0-common]
+   [metabase-enterprise.representations.v0.database :as v0-database]
+   [metabase-enterprise.representations.v0.document :as v0-document]
+   [metabase-enterprise.representations.v0.metric :as v0-metric]
+   [metabase-enterprise.representations.v0.model :as v0-model]
+   [metabase-enterprise.representations.v0.question :as v0-question]
+   [metabase-enterprise.representations.v0.snippet :as v0-snippet]
+   [metabase-enterprise.representations.v0.transform :as v0-transform]
+   [toucan2.core :as t2]))
+
+(defn toucan-model
+  "Returns the Toucan model keyword for a given v0 representation type.
+   Maps representation types (e.g., :collection, :database) to their corresponding Toucan models."
+  [representation-type]
+  (case representation-type
+    :collection v0-coll/toucan-model
```

Comment:
wouldn't it be easier just to make these all one `toucan-model` multimethod, where `:collection` and the like is the dispatch value?